### ISSUE TITLE
How is the security system secured from attackers?

### ISSUE THREAD

**Issue by MitchellJThomas** [opencybersecurityalliance/documentation#15]
Who or what is securing the security system? 
What happens when malware is introduced into this system?  If it's connected to everything that seems like a great vector to create a lot of damage.

Idea:  Borrow context and components from [NIST's Zero Trust Architecture](https://www.nist.gov/publications/zero-trust-architecture).  Say assume an identity provider, user/role based encryption, [message authentication codes](https://en.wikipedia.org/wiki/Message_authentication_code) and risk posture system that allows for self assessment to occur.

**Comment by warrenrjwc** [comment]
What is the intention of this suggestion?  I am not clear on what we want to use this for.  Is this a suggested use case?

**Comment by MitchellJThomas** [comment]
My intention is to create discussion about establishing and maintaining trust in a system critical to operations, mainly to reach clarity around whether the architecture provides or requires those trust components.  Ignoring this we run the risk of designing a vulnerable system.

**Comment by adammontville** [comment]
It's a good idea to look at this.

**Comment by JasonKeirstead** [comment]
I think we need to be a bit careful about getting into the weeds here. Not all cybersecurity operation stacks run in a zero trust model nor would we want to suggest that should even be required.... it would immediately limit the applicability of our work to only those organizations who are embracing such a model, which are still somewhat rare.

**Comment by adammontville** [comment]
I agree that we don't want to be so forward looking that we lose the ability to transition. Perhaps there's still something we can glean from looking at a zero trust architecture. Are there decisions we could make to day that would hinder embracing that model in the future? Are there base components in a zero trust architecture that are prevalent enough today to consider including/assuming?

**Comment by sparrell** [comment]
I agree we should be looking at the question "how do you secure OCA systems?" As the original point made is valid that security systems are prime target. I agree with the comment that there are different ways to do this and different needs for different use cases and different legacy architectures. I recommend against getting too prescriptive too soon. We might want to start by gathering examples of how you could do OCA system security. And we should look at all aspects. Personally I'd start with some recursive aspects of the system feeding itself especially in any proof of concept. E.g. drink our own champagne
